cmake/scripts/windowsstore/ArchSetup.cmake

```diff
@@ -1,12 +1,15 @@
-# Minimum SDK version we support
-set(VS_MINIMUM_SDK_VERSION 10.0.22621.0)
+# Minimum SDK version required to build
+set(VS_MINIMUM_BUILD_SDK_VERSION 10.0.22621.0)
 
-if(CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION VERSION_LESS VS_MINIMUM_SDK_VERSION)
+if(CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION VERSION_LESS VS_MINIMUM_BUILD_SDK_VERSION)
   message(FATAL_ERROR "Detected Windows SDK version is ${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION}.\n"
-    "Windows SDK ${VS_MINIMUM_SDK_VERSION} or higher is required.\n"
+    "Windows SDK ${VS_MINIMUM_BUILD_SDK_VERSION} or higher is required.\n"
     "INFO: Windows SDKs can be installed from the Visual Studio installer.")
 endif()
 
+# Minimum OS version to run the app
+set(VS_MINIMUM_SDK_VERSION 10.0.18362.0)
```

Comment:
Only a personal preference but I prefer the two variables on top as are more visible. It's the only thing that usually needs to be modified on these scripts.

Maybe change order is enough:
```
# Minimum SDK version required to build
set(VS_MINIMUM_BUILD_SDK_VERSION 10.0.22621.0)
# Minimum OS version to run the app
set(VS_MINIMUM_SDK_VERSION 10.0.18362.0)
```

Reply:
sure, doesn't really matter to me. hopefully this time all builds are green.